wallet/src/Pos/Wallet/Web/Tracking/Sync.hs

```diff
@@ -493,19 +481,19 @@ getEncInfo encSK = do
     let wCId = addressToCId $ makeRootPubKeyAddress pubKey
     (hdPass, wCId)
 
-selectOwnAccounts
+selectOwnAddresses
     :: (HDPassphrase, CId Wal)
     -> (a -> Address)
     -> [a]
     -> [(a, CWAddressMeta)]
-selectOwnAccounts encInfo getAddr =
-    mapMaybe (\a -> (a,) <$> decryptAccount encInfo (getAddr a))
+selectOwnAddresses encInfo getAddr =
+    mapMaybe (\a -> (a,) <$> decryptAddress encInfo (getAddr a))
 
 setLogger :: HasLoggerName m => m a -> m a
 setLogger = modifyLoggerName (<> "wallet" <> "sync")
 
-decryptAccount :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
-decryptAccount (hdPass, wCId) addr = do
+decryptAddress :: (HDPassphrase, CId Wal) -> Address -> Maybe CWAddressMeta
+decryptAddress (hdPass, wCId) addr = do
```

Comment:
Yep, all the changes tracked here comes from [here](https://github.com/input-output-hk/cardano-sl/commit/fee01810a33d73ebf349a0995efc0dd74ced6f58).